webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require('webpack');
+
+var PLUGINS = [];
+if (process.env.NODE_ENV === 'production') {
+  new webpack.optimize.UglifyJsPlugin()
```

Comment:
If you're trying to add this to the `PLUGINS` array, you might want to do something like this:

```javascript
PLUGINS.push(new webpack.optimize.UglifyJsPlugin());
```